aten/src/ATen/native/mkldnn/xpu/detail/Attr.h

```diff
@@ -371,4 +367,57 @@ class Attr {
   dnnl::post_ops dnnl_post_ops_;
 };
 
+static inline void construct_attr_for_unary(
+    const c10::string_view& unary_post_op,
+    const torch::List<std::optional<at::Scalar>>& unary_post_op_args,
+    const c10::string_view& unary_post_op_algorithm,
+    at::native::onednn::Attr& attr) {
+  if (unary_post_op == "relu") {
+    attr = attr.append_post_eltwise(
+        /* eltwise_scale */ 1.f,
+        /* alpha */ 0.f,
+        /* beta */ 0.f,
+        attr.kind_with_relu);
+  } else if (unary_post_op == "leaky_relu") {
+    auto alpha = unary_post_op_args[0].value().to<float>();
+    attr = attr.append_post_eltwise(1.0, alpha, 0.f, attr.kind_with_relu);
+  } else if (unary_post_op == "tanh") {
+    attr = attr.append_post_eltwise(1.0f, 0.0f, 0.0f, attr.kind_with_tanh);
+  } else if (unary_post_op == "gelu") {
+    auto post_algorithm = unary_post_op_algorithm == "none"
+        ? attr.kind_with_gelu_erf
+        : attr.kind_with_gelu_tanh;
+    attr = attr.append_post_eltwise(1.0f, 0.0f, 0.0f, post_algorithm);
+  } else if (unary_post_op == "hardtanh") {
+    auto alpha = unary_post_op_args[0].value().to<float>();
+    auto beta = unary_post_op_args[1].value().to<float>();
+    attr = attr.append_post_eltwise(1.0, alpha, beta, attr.kind_with_clip);
+  } else if (unary_post_op == "hardswish") {
+    attr = attr.append_post_eltwise(
+        1.0f, 1.f / 6.f, 1.f / 2.f, attr.kind_with_hardswish);
+  } else if (unary_post_op == "swish") {
+    attr = attr.append_post_eltwise(1.0f, 1.0f, 0.0f, attr.kind_with_swish);
+  } else {
+    TORCH_CHECK(
+        unary_post_op == "none",
+        "onednn qlinear: unspported unary post op",
+        unary_post_op);
+  }
+}
+
+static inline void construct_attr_by_post_op(
+    const c10::string_view& binary_post_op,
+    double binary_alpha,
+    double input1_scale,
+    int64_t input1_zero_point,
+    const c10::string_view& unary_post_op,
+    const torch::List<std::optional<at::Scalar>>& unary_post_op_args,
+    const c10::string_view& unary_post_op_algorithm,
+    at::native::onednn::Attr& attr) {
+  if (binary_post_op == "none") {
```

Comment:
I think it is impossible to enable `binary_post_op` and `unary_post_op` at the same time. So, it is better to add assert.
```suggestion
  TORCH_INTERNAL_ASSERT(!(binary_post_op != "none" && unary_post_ops != "none"));
```

Reply:
We can support binary+unary like `conv_add_relu`.

Reply:
So, I would suggest something like:

```c++
bool is_binary_post_op_only = (binary_post_op != "none" && unary_post_op == "none"); // ex., conv + add
bool is_unary_post_op_only = (binary_post_op == "none" && unary_post_op != "none"); // ex., conv + relu
bool is_binary_and_unary_post_op = (binary_post_op != "none" && unary_post_op != "none"); // ex., conv + add + relu
TORCH_INTERNAL_ASSERT(is_unary_post_op_only, "Currently, quantization backend for Intel GPU only supports the fusion of Convolution and unary operation like ReLU");
```

Reply:
```
bool is_none_post_op = (binary_post_op == "none" && unary_post_op == "none");  // not post-ops
TORCH_INTERNAL_ASSERT(is_unary_post_op_only || is_none_post_op, "Currently, quantization backend for Intel GPU only supports convolution or convolution with unary post operation like ReLU");
```
Add a case for describing no post-ops case.